draft-ietf-sedate-datetime-extended.md

```diff
@@ -369,7 +369,7 @@ in email headers, described in {{Section 3.3 of RFC5322}} and introduced
 earlier in {{Section 3.3 of RFC2822}}.
 This email header convention is in actual use, while its adaptation into
 {{RFC3339}} always was
-handicapped by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
+held back by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
```

Comment:
```suggestion
severely undermined by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
```

I think this should do the trick :smile:

Reply:
> I think this should do the trick 😄

short and sweet:

```suggestion
compromised by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
```